index.html

```diff
@@ -15,7 +15,7 @@
 		<div id='controls'>
 			<form onsubmit='search(); return false;'>
 				<input type='search' autocomplete='off' id='search-term' placeholder='Search...' autofocus='' />
-				<input type='submit' value='Search' />
+				<input id="searchButton" type='submit' value='Search' disabled="disabled"/>
```

Comment:
I'd prefer not making changes in the formatting before the linting and refactoring PRs are merged, otherwise those changes will be all over the place, and the code will be in inconsistent state until then.

Reply:
I didn't think of that, thanks for pointing it out